algo/src/test/java/com/hartwig/serve/sources/ckb/ActionableTrialFactoryTest.java

```diff
@@ -0,0 +1,147 @@
+package com.hartwig.serve.sources.ckb;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Set;
+
+import com.google.common.collect.Lists;
+import com.hartwig.serve.ckb.datamodel.CkbEntry;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.ClinicalTrial;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.ImmutableLocation;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.ImmutableVariantRequirementDetail;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.Location;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.VariantRequirementDetail;
+import com.hartwig.serve.datamodel.EvidenceDirection;
+import com.hartwig.serve.datamodel.EvidenceLevel;
+import com.hartwig.serve.datamodel.Knowledgebase;
+
+import org.apache.logging.log4j.util.Strings;
+import org.junit.Test;
+
+public class ActionableTrialFactoryTest {
+
+    @Test
+    public void canCreateActionableTrials() {
+        Location location = ImmutableLocation.builder().nctId("").city("").country("United States").status("Recruiting").build();
+        VariantRequirementDetail requirementType =
+                ImmutableVariantRequirementDetail.builder().profileId(0).requirementType("required").build();
+        CkbEntry entryDeletion = CkbTestFactory.createEntryWithClinicalTrial("KRAS",
+                "deletion",
+                "KRAS deletion",
+                "sensitive",
+                "Emerging",
+                "AB",
+                "AB",
+                "A",
+                "DOID:162",
+                location,
+                "Recruiting",
+                requirementType);
```

Comment:
I would make a simpler utility function. For trials you don't care about evidence so that can be omitted from this utility function.

Reply:
I've removed the evidence stuff from the ActionableTrialFactoryTest